### PR TITLE
[pl]: 8 keys updated

### PR DESCRIPTION
**Pull request by makowskid**

### CODE REVIEW

File: locales/pl/php-inline.json [hunk 1]
```diff
@@ -92,10 +92,10 @@
     "password.symbols": "To pole musi zawierać przynajmniej jeden symbol.",
     "password.uncompromised": "Podane pole pojawiło się w wycieku danych. Proszę wybrać inną wartość pola.",
     "present": "Pole musi być obecne.",
-    "present_if": "This field must be present when :other is :value.",
-    "present_unless": "This field must be present unless :other is :value.",
-    "present_with": "This field must be present when :values is present.",
-    "present_with_all": "This field must be present when :values are present.",
+    "present_if": "Pole :attribute musi być obecne jeżeli :other ma wartość :value.",
```
Comment (andrey-helldar) [inline locales/pl/php-inline.json:95]:
```suggestion
    "present_if": "Podane pole musi być obecne jeżeli :other ma wartość :value.",
```
And for the remaining keys you also need to change the message format.

**Review by andrey-helldar — CHANGES_REQUESTED** [review]

File: locales/pl/php-inline.json [hunk 2]
```diff
@@ -92,10 +92,10 @@
...
+    "present_unless": "Pole :attribute musi być obecne, chyba że :other ma wartość :value.",
+    "present_with": "Pole :attribute musi być obecne kiedy :values jest podany/a..",
+    "present_with_all": "Pole :attribute musi być obecne kiedy :values są podane.",
```
Comment (andrey-helldar) [inline locales/pl/php-inline.json:98]:
```suggestion
    "present_if": "Pole musi być obecne jeżeli :other ma wartość :value.",
    "present_unless": "Pole musi być obecne, chyba że :other ma wartość :value.",
    "present_with": "Pole musi być obecne kiedy :values jest podany/a..",
    "present_with_all": "Pole musi być obecne kiedy :values są podane.",
```
(I don't know if it's right or not, just did it like the other keys.)

**Review by andrey-helldar — CHANGES_REQUESTED** [review]

**Comment by makowskid** [comment]
@andrey-helldar fixed

**Review by andrey-helldar — APPROVED** [review]
Dziękuję

**Pull request merged**